address.js

```diff
@@ -0,0 +1,57 @@
+const Router = require("koa-router");
+
+const router = new Router();
+
+const Web3 = require("web3");
+
+const web3 = new Web3(
+  new Web3.providers.HttpProvider(
+    "https://kovan.infura.io/v3/3269c013c5b2449aaea1bb593f873d77"
+  )
+);
+
+const contractAddress = "0xf019a9c291fab47003588fe74d33660cd467fc22";
+const abi = require("./abi.js");
+const contract = new web3.eth.Contract(abi, contractAddress);
+
+
+class ResponseData {
```

Comment:
We follow the functional programming paradigm; so we don't really need classes. In your case all you needs is an object

```
const createResponseData = (from, to, value) => ({from , to, value})

```